runtime/src/parachain.rs

```diff
@@ -24,9 +24,9 @@ macro_rules! runtime_parachain {
 		pub const VERSION: RuntimeVersion = RuntimeVersion {
 			spec_name: create_runtime_str!("moonbase-alphanet"),
 			impl_name: create_runtime_str!("moonbase-alphanet"),
-			authoring_version: 2,
-			spec_version: 2,
-			impl_version: 3,
+			authoring_version: 3,
+			spec_version: 3,
+			impl_version: 0,
```

Comment:
@JoshOrndorff This should be 1 no ?

Reply:
I believe the versioning convention is set it to 0 on `spec_version` bump, but I may be wrong.

Reply:
Looks like Polkadot uses zero https://github.com/paritytech/polkadot/blob/355558ae880b6f2d256550f02fc41f680b0f752b/runtime/polkadot/src/lib.rs#L97

(I also thought it was 1, but I guess not)